translations/management/commands/bulk_translate.py

```
@@ -1,5 +1,6 @@
 from django.core.management.base import BaseCommand
 from django.conf import settings
+from django.db.models import Q, OuterRef, Exists
```

Ruff (0.12.2)

`django.db.models.Q` imported but unused

Remove unused import

(F401)